data/wanderer/wanderers start.txt

```diff
@@ -179,67 +183,390 @@ mission "Wanderers: Hai Diplomat"
-	destination "Vara K'chrai"
 	clearance
 	passengers 1
 	to offer
 		has "First Contact: Wanderer: offered"
+		not "event: hai-human resolution announced"
+	to complete
+		or
+			has "Wanderers: Diplomacy: offered"
+			has "Wanderers: Diplomacy (Before Hai Reveal): done"
 	on offer
-		log "People" "Sayari" `Now that the crisis with humanity is over she has volunteered to serve as a translator and be the Hai diplomat to the Wanderers as a capstone for her career.`
 		conversation
-			`As you are approaching Hai-home, you contact the Hai government and inform them that you have found new aliens to the north. You submit a request for ambassadorial assistance to help you communicate with the aliens to the north, who seem to speak the Hai language. They immediately direct you to a now familiar private government hangar, where a familiar diplomat meets you.`
-			`	"Good to see you again Captain <last>, I have arranged for some refreshments so please, come and tell me all about these new aliens you have discovered!"`
-			`	Her exuberance is catching and after recent stressful events you see a new side to her as you relay your experience with the bird-like aliens. You realize that opportunities like this are why she became a diplomat in the first place and that this is, to some degree, a dream come true.`
-			`	"So in conclusion, it seems you need a translator," she says with a wry smile.`
+			action
+				event "A Hai professor is available to translate" 7
+			`As you approach Hai-home, you contact the Hai government and inform them that you have found new aliens to the north. You submit a request for ambassadorial assistance to help you communicate with the aliens, who seem to speak the Hai language. They immediately direct you to a government hangar in a compound called "Directorate of Intra-Hai Affairs." A Hai approaches you in an unfamiliar uniform.`
+			# Hai Intelligence officer subtly suggests they already know what's happening, and would have done
+			# something about by now if it were an opportune time.
+			`	"I received word you met some troubled avians to the north. What is your intention?"`
+			choice
+				`	"They speak the Hai language, and I need an interpreter."`
+					goto response
+				`	"I'd like to make first contact with their species."`
+					goto response
+				`	"Wait a minute. I never mentioned they were avians. Why do you say they're troubled?"`
+			`	The uniformed Hai ignores your question and asks again, "What is your intention, Captain <last>?"`
+			choice
+				`	"They speak the Hai language, and I need an interpreter."`
+				`	"I'd like to make first contact with their species."`
+			label response
+			branch election
+				or
+					has "Hai Reveal [A11] Sound the Alarm: active"
+					has "Hai Reveal [A12] Into the Lion's Den: active"
+			branch stagnant
+				not "Hai Reveal [A11] Sound the Alarm: offered"
+			# We get here if the election has ended, but the Hai-human resolution hasn't been announced
+			branch chaos
```

Comment:
```suggestion
			branch chaos
				1 == 1
```

Reply:
That is not necessary. `branch somewhere` with no condition is an unconditional branch.

Reply:
~~How am I just now learning about this~~

Reply:
I think adding a redundant comparison makes it more confusing than saying, "Yeah, we're just branching into chaos."

Reply:
Yeah, this can be resolved